pkg/kubelet/kubelet.go

```diff
@@ -1344,7 +1336,7 @@ func (kl *Kubelet) Run(updates <-chan kubetypes.PodUpdate) {
 
 	if kl.kubeClient != nil {
 		// Start syncing node status immediately, this may set up things the runtime needs to run.
-		go wait.Until(kl.syncNodeStatus, kl.nodeStatusUpdateFrequency, wait.NeverStop)
+		go wait.Until(kl.nodeStatusManager.SyncNodeStatus, kl.nodeStatusUpdateFrequency, wait.NeverStop)
```

Comment:
Not mandatory, but I would go one step further and put the retry logic inside a Start() func in the node status manager, including the fastStatusUpdateOnce function.

Reply:
done